httpserver-rs/Cargo.toml

```diff
@@ -0,0 +1,47 @@
+[package]
+name = "wasmcloud-provider-httpserver"
+version = "0.1.2"
+edition = "2018"
+resolver = "2"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+async-trait = "0.1"
+base64 = "0.13"
+bytes = "1.0"
+chrono = "0.4"
+crossbeam="0.8"
+env_logger = "0.9"
+futures = "0.3"
+http = "0.2"
+log = "0.4"
+once_cell = "1.8"
+rmp-serde = "0.15"
+serde_bytes = "0.11"
+serde_json = "1.0"
+serde = {version = "1.0", features = ["derive"] }
+thiserror = "1.0"
+tokio = { version = "1", features = ["full"] }
+toml = "0.5"
+warp = {version="0.3", features=["tls"]}
+
+[dependencies.wasmcloud-interface-httpserver]
```

Comment:
convert this to a regular dependency declaration, e.g. `wasmcloud-interface-httpserver = "0.1"`